thrift-exemplo/java/src/main/java/exemplo/api/ExemploEnvioApi.java

```diff
@@ -0,0 +1,87 @@
+package exemplo.api;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+import org.apache.hc.client5.http.classic.methods.HttpPost;
+import org.apache.hc.client5.http.entity.mime.MultipartEntityBuilder;
+import org.apache.hc.client5.http.impl.classic.CloseableHttpClient;
+import org.apache.hc.client5.http.impl.classic.HttpClients;
+import org.apache.hc.core5.http.ContentType;
+import org.apache.hc.core5.http.HttpStatus;
+import org.apache.hc.core5.http.io.entity.EntityUtils;
+
+import br.gov.saude.esus.transport.common.generated.thrift.DadoTransporteThrift;
+
+import exemplo.ExemploFichaProcedimentoThrift;
+import exemplo.common.SerializadorThrift;
+
+public class ExemploEnvioApi {
+
+	public static void main(String[] args) throws IOException {
+
+		// Passo 1: Setar os valores nas variáveis APPLICATION_URL, USUARIO_CREDENCIAL e SENHA_CREDENCIAL abaixo para configurar o envio
+		String APPLICATION_URL = "COLOQUE_AQUI_URL_SERVIDOR_PEC";  /* Endereço de ip da instalação a se fazer o envio */
+		String LOGIN_URL = APPLICATION_URL + "/api/recebimento/login";
+		String ENVIO_URL = APPLICATION_URL + "/api/v1/recebimento/ficha";
+		String USUARIO_CREDENCIAL = "COLOQUE_AQUI_USUARIO";    /* Usuário da credencial obtida com o responsãvel pelo recebimento. */
+		String SENHA_CREDENCIAL = "COLOQUE_AQUI_SENHA";    /* Senha da credencial obtida com o responsãvel pelo recebimento. */
```

Comment:
Apenas pra organizar o que deve ser alterado ficar em primeiro
```suggestion
		// Passo 1: Setar os valores nas variáveis USUARIO_CREDENCIAL, SENHA_CREDENCIAL e APPLICATION_URL para configurar o envio
		String USUARIO_CREDENCIAL = "COLOQUE_AQUI_USUARIO";    /* Usuário da credencial obtida com o responsãvel pelo recebimento. */
		String SENHA_CREDENCIAL = "COLOQUE_AQUI_SENHA";    /* Senha da credencial obtida com o responsãvel pelo recebimento. */
		String APPLICATION_URL = "COLOQUE_AQUI_URL_SERVIDOR_PEC";  /* Endereço de ip da instalação a se fazer o envio */
		String LOGIN_URL = APPLICATION_URL + "/api/recebimento/login";
		String ENVIO_URL = APPLICATION_URL + "/api/v1/recebimento/ficha";
```